runtime/src/bank.rs

```diff
@@ -1974,6 +1973,7 @@ impl Bank {
             self.freeze_started.store(true, Relaxed);
             *hash = self.hash_internal_state();
         }
+        self.rc.accounts.accounts_db.mark_slot_frozen(self.slot());
```

Comment:
maybe put this side the above if `*hash == Hash::default()`? `fn freeze` can be called multiple times and it'll trigger multiple mertrics reporting in the `mark_slot_frozen`?

Reply:
makes sense, done!